src/services/profile.py

```diff
@@ -45,6 +45,12 @@
             "text": u'🙋🏽 ආයුබෝවන් {{user_first_name}}! විවෘත මූලාශ්‍ර 🔓 වලට'
             ' අදාල Facebook DevC වලින් ලබාදෙන සම්පත් වලට ප්‍රවේශ වෙන්න,'
             ' Get Started බටන් එක ක්ලික් කරන්න.'
+        },
+        {
+            "locale": "rw_RW",
+            "text": u'🙋🏽 Muraho {{user_first_name}}! Kanda kuri Buto Tangira '
+            'hepfo kugirango ugere kuri Facebook DevC ibikoresho byegeranyijwe bijyanye'
```

Comment:
@Emile-Nsengimana before we merge you PR please correct this simple [issue](https://github.com/fbdevelopercircles/open-source-edu-bot/pull/99/checks): Line 52 should be broken, it is too long.

```bash
============================= test session starts ==============================
platform linux -- Python 3.7.8, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
rootdir: /app, inifile: setup.cfg, testpaths: tests
collected 6 items

tests/test_factory.py .                                                  [ 16%]
tests/test_services_messenger.py ....                                    [ 83%]
tests/test_services_profile.py .                                         [100%]

============================== 6 passed in 0.35s ===============================
./services/profile.py:52:80: E501 line too long (88 > 79 characters)
##[error]Process completed with exit code 1.
```

Reply:
done @elinguiuriel